docs/UserGuide.md

```diff
@@ -263,8 +263,58 @@ There must not be duplicate fields. For example, if there is a contact with the
    * **Fail:** `n/Jane Hoe th/janehoe e/janeh@example.com ss/undergraduate 3 r/Admin r/President nn/jane`
 
 
-## Available fields
-
+## Available Contact Fields
+
+| Field                                   | Prefix | Valid examples
+|-----------------------------------------|--------|---------------
+| [**NAME**](#name)                       | `n/`   | `Gina Tan`<br/> `Jane Smith @ Jones`<br/> `Ravi S/O Ramasamy`<br/> `Devi D/O Rajaratnam`<br/> `Janelle Wong (Jane)`
+| [**TELEGRAM HANDLE**](#telegram-handle) | `th/`  | `ginatan123`<br/> `jane_smith28` 
+| [**EMAIL**](#email)                     | `e/`   | `gina_tan@example-website.com.sg`
+| [**STUDENT STATUS**](#student-status)   | `ss/`  | `Undergraduate 3`<br/> `Masters`<br/> `PhD`<br/> 
+| [**ROLE**](#role)                       | `r/`   | `President`<br/> `Events (External)`
+| [**NICKNAME**](#nickname)               | `nn/`  | `genie34 ;)`
+
+### Name
+- Should contain alphabets and spaces only (with exceptions).
```

Comment:
Should contain --> Contains

(with exceptions) --> (with only the following exceptions)